CRM/Case/BAO/Case.php

```diff
@@ -1212,9 +1212,20 @@ public static function getCaseActivity($caseID, &$params, $contactID, $context =
    *
    */
   public static function getRelatedContacts($caseID, $skipDetails = FALSE) {
+    $caseRoles = array();
+    if (!$skipDetails) {
+      $caseInfo = civicrm_api('Case', 'getsingle', array(
```

Comment:
Ok I found the problem

_civicrm_api3_case_read 

calls getRelatedContacts causing a code loop & tests to fail. Defaulting to not getting the additional data will fix this (as well as reduce overhead - per comment later)